android/app/build.gradle

```diff
@@ -396,8 +396,8 @@ dependencies {
     // from https://github.com/getlantern/opus_android
     implementation files('libs/opuslib-release.aar')
     implementation 'com.github.getlantern:secrets-android:f6a7a69f3d'
-    implementation 'com.github.getlantern:db-android:5a082e0bdd'
-    implementation 'com.github.getlantern:messaging-android:ca369d5173'
+    implementation 'com.github.getlantern:db-android:2e81ba6542'
+    implementation 'com.github.getlantern:messaging-android:3f23f07b4e'
```

Comment:
Also, we need to update android-db on messages as well Unless it will rollback to an older version.